app/importers/concerns/taxonomy_methods.rb

```diff
@@ -0,0 +1,35 @@
+module TaxonomyMethods
+  attr_accessor :taxonomy_parser
+
+  def get_missing_country(mappable_field)
+    country_array = normalize_industry('Missing Country: ' + mappable_field)
+    country_array.nil? ? nil : lookup_country(country_array.first)
+  end
+
+  def set_taxonomy_parser
+    self.taxonomy_parser = TaxonomyParser.new(Rails.configuration.frozen_protege_source)
+    taxonomy_parser.concepts = YAML.load_file(Rails.configuration.frozen_taxonomy_concepts)
+  end
```

Comment:
Good call on the lazy instantiation, that is much, much cleaner.  

For your second point, I think either way makes sense.  When I did the gem originally, I didn't plan for the concepts to be set at initialization time, rather they're set by the parser after calling the parse method (which is actually where these concepts come from... see lib/tasks/taxonomy.rake).  This was more of a workaround for performance reasons.